packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -831,23 +913,30 @@ void main() {
         ),
       ),
     );
-
-    expect(tester.getSemantics(find.byKey(expandedKey)), matchesSemantics(
-      label: 'Expanded',
+    final Finder expandedIcon = find.descendant(
+      of: find.ancestor(
+        of: find.byKey(expandedKey),
+        matching: find.byType(Row),
+      ),
+      matching: find.byType(MergeSemantics),
+    );
+    expect(tester.getSemantics(expandedIcon), matchesSemantics(
       isButton: true,
       hasEnabledState: true,
       isEnabled: true,
       hasTapAction: true,
       onTapHint: localizations.expandedIconTapHint,
     ));
 
-    expect(tester.getSemantics(find.byKey(collapsedKey)), matchesSemantics(
+    final Finder collapsedHeader = find.ancestor(
+      of: find.byKey(collapsedKey),
```

Comment:
How did the semantics tree change that collapsedKey doesn't get us to the right semantics node anymore?